Makefile

```diff
@@ -1,16 +1,17 @@
 STACK_NAME ?= ffmpeg-lambda-layer
+DEPLOYMENT_PREFIX ?= ""
 
-clean: 
+clean:
```

Comment:
Looks as though some whitespace was trimmed by my editor; let me know if I should revert these.